### ISSUE TITLE
RSPEC-5734 Allowing browsers to sniff MIME types is security-sensitive

### ISSUE THREAD

**Issue by eric-therond-sonarsource** [SonarSource/SonarJS#1953]
Implements [RSPEC-5734](https://jira.sonarsource.com/browse/RSPEC-5734) MMF-1898

For both JS/TS

Code examples: https://github.com/SonarSource/security-expected-issues/tree/master/javascript/express.js/RSPEC-5734

Should raise on the `noSniff` property of the argument of an `helmet()` call when set to `false`:

```
  app.use(
    helmet({
      noSniff: false, // Sensitive
    })
  );
```

**Comment by eric-therond-sonarsource** [comment]
Reopened because like for issues the message is incorrect/not formatted ([see this issue for instance](https://peach.sonarsource.com/security_hotspots?id=security-expected-issues-express-js_nodev8&hotspots=AXWxbicoMh1ExG5XrU4i))
